extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClients.java

```diff
@@ -249,6 +250,8 @@ private MongoClientSettings createMongoConfiguration(MongoClientConfig config) {
                 CodecRegistries.fromProviders(providers));
         settings.codecRegistry(registry);
 
+        settings.commandListenerList(getCommandListeners(mongoClientSupport.getCommandListeners()));
```

Comment:
Maybe call this one `commandListeners`?

Reply:
I'm not sure about this suggestion.
- The settings class is from mongodb driver (com.mongodb.MongoClientSettings.Builder)
- The MongoClientSupport class has all its fields exposed to getter methods
- The private method getCommandListeners is called after the same naming pattern as the getCodeProviders method.

It could be ... extracting the call to the getter function to a variable called commandListeners (very likely to happen in a future PR). For example: 

```
List<CommandListeners> commandListeners = getCommandListeners(mongoClientSupport.getCommandListeners());
settings.commandListenerList(commandListeners);
```